### PR TITLE
add active id to accordion change event

### LINKED ISSUE #6130: fix: custom 'change' event has no arguments in fast-accordion
# 🐛 Bug Report

The Accordion component fires a custom 'change' event when the active item changes however the event does not contain any data on what item is affected/clicked.

## 🤔 Expected Behavior

I would expect a behavior where the affected fast-accordionitem (id) would be part of the event arguments like it is with the fast-tabs component

## 😯 Current Behavior

The 'change' event is empty/has no parameters

## 💁 Possible Solution

Add the (already available) private activeid variable to the event parameters

## 🔦 Context

Having this event argument available would allow us to create a binding on the Fluent UI Blazor component so that clicking on an item can be captured 

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

### PR DESCRIPTION
**Pull request by chrisdholt**
# Pull Request

## 📖 Description
Adds information on the activeid for the accordion as part of the elements custom event.

### 🎫 Issues
closes #6130 

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by EisenbergEffect — COMMENTED** [review]
Is this consistent with the way we handle it in other components? I don't recall seeing our code using the id in the event args before.

**Comment by chrisdholt** [comment]
> Is this consistent with the way we handle it in other components? I don't recall seeing our code using the id in the event args before.

As @vnbaaij calls out, tabs does this exact thing. The delta here from other components is that activeid is internal to accordion and private, so passing it as part of the event should allow us to expose it gracefully.

File: change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "add activeid to accordion change event",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "chhol@microsoft.com",
+  "dependentChangeType": "patch"
```
Comment (EisenbergEffect) [inline change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json:6]: I think we need to change this to "prerelease" also.

**Review by EisenbergEffect — APPROVED** [review]
Probably need to port this to the archive branch as well.

**Comment by vnbaaij** [comment]
After looking at the source again, I see the accordion-item emits the `change` event too. And that one has a public `id`. As he events bubble up (if I understood correctly), maybe it makes more sense to change it in the accordion-item itself?

**Comment by chrisdholt** [comment]
> After looking at the source again, I see the accordion-item emits the `change` event too. And that one has a public `id`. As he events bubble up (if I understood correctly), maybe it makes more sense to change it in the accordion-item itself?

Eh, we could, but I wonder if that's enough in and of itself - that value is already available publicly in the event object I believe.

File: change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json (hunk 1 above)
Comment (chrisdholt) [reply change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json:6]: yep, thanks

**Comment by scomea** [comment]
> > After looking at the source again, I see the accordion-item emits the `change` event too. And that one has a public `id`. As he events bubble up (if I understood correctly), maybe it makes more sense to change it in the accordion-item itself?

This is related.  https://github.com/microsoft/fast/issues/6144 to "accordion" reacting to "change" events from items other than "accordion-item" .  I have a local branch that adds checks to "accordion" to ensure it only reacts to these events from direct children but I do wonder if we should have more specific event names rather than so many "change" events (ie. "expanded").

File: change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json (hunk 1 above)
Comment (chrisdholt) [reply change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json:6]: Dangit - I missed pushing this...PR incoming

**Pull request merged**